### ISSUE TITLE
Ccompat API should accept both application/vnd.schemaregistry.v1+json and application/json

### ISSUE THREAD

**Issue by EricWittmann** [Apicurio/apicurio-registry#1449]
Our ccompat API currently accepts `application/vnd.schemaregistry.v1+json` as the input/output for all JSON based requests.  So if a user tries to make a request using `Accept: application/json` (for example) the request will fail.  We should allow both.

**Comment by EricWittmann** [comment]
Note: this problem has been reported for version 1.3.2.Final.  The report includes this reproducer:

```
curl -vi -H "Accept: application/json" -H "Content-Type: application/json" http://reg-service-54hv5.schema-registry:8080/api/ccompat/subjects
```

That will apparently fail.  I have not yet tried it to confirm.

**Comment by EricWittmann** [comment]
It's possible that the problem is adding an `Accept` header for a GET operation...

**Comment by carlesarnal** [comment]
This was reported in the past and has been fixed for versions later than 1.3.2.Final, see #1017.

**Comment by EricWittmann** [comment]
Yes sorry - closing as duplicate of https://github.com/Apicurio/apicurio-registry/issues/1007.
